docs/connected_orgs.rst

```diff
@@ -1,14 +1,95 @@
-Connect persistent orgs
+Connect Persistent Orgs
 =======================
+In order to connect persistent orgs such as a Developer Edition, Enterprise Edition, or Sandbox org to CumulusCI, you first need to have a Connected App configured in a persistent Salesforce org.
 
-Connect a production org
-------------------------
 
-Connect a sandbox org
+
+CumulusCI's Connected App
```

Comment:
Can we avoid starting with Connected Apps and reserve it for the end section? I don't think we want to further the occasional confusion that users _need_ to create Connected Apps.